pdf_map_export/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM camptocamp/mapfish_print:latest
+FROM camptocamp/mapfish_print:3.28
```

Comment:
Warum nicht eine fixe Version? --> klar, mühsamer zum updaten & up-to-date halten. aber dafür keine "breaking-changes" welche automatisch einfliessen

Reply:
Gut gesehen, die meisten anderen dependencies sind ebenfalls auf einer fixen Version